internal/service/wafv2/flex.go

```diff
@@ -2309,7 +2309,7 @@ func flattenBodyContains(apiObject *wafv2.ResponseInspectionBodyContains) []inte
 
 	m := map[string]interface{}{
 		"failure_strings": flex.FlattenStringSet(apiObject.FailureStrings),
-		"succeed_strings": flex.FlattenStringSet(apiObject.SuccessStrings),
```

Comment:
Were you able to add an Account Takeover Protection rule to a WAFv2 Web ACL using `success_strings` instead of `succeed_strings` parameter? In #31106 issue I pasted the errors I get in this case, and I was able to add an ATP rule using `succeed_strings` successfully.

Reply:
Yes, these lines in the `flatten...` functions were the problem. They should have been `success_...` instead of `succeed_...` to match the names of the parameters returned by AWS